### PR TITLE
feat(scss&qa) tabs bars

### LINKED ISSUE #1356: [DS to LF] Navigation : Tabs bars
- [x] Base color (grey 700)
- [x] Creation mod-small
- [x] Item selected (primary 700, font-weight 400)
- [x] Add mod-number
- [x] Spacings to be revised
- [x] Create mod without border-bottom
- [x] Add disabled state

Capture d’écran 2021-05-26 à 17 45 08

### PR DESCRIPTION
**Pull request by AlexiaAlarza**
fixes #1356

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1423

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1423

File: stories/qa/menu/menu.stories.html [hunk 1]
```diff
@@ -2,29 +2,60 @@
 <section class="contentSection">
 	<h1>Menu</h1>
 	<nav class="menu">
-		<a class="menu-link" aria-current="page">
-			active link
-			<span class="menu-link-placeholder" aria-hidden="true" data-content-after="active link"></span>
+		<a class="menu-link is-active">active link</a>
```
Comment (MarcFairbrother) [inline stories/qa/menu/menu.stories.html:5]: Shouldn't we keep the `aria-current="page"` attribute for accessibility?

File: stories/qa/menu/menu.stories.html [hunk 2]
```diff
@@ -2,29 +2,60 @@
...
+		<a class="menu-link">menu link</a>
+		<a class="menu-link">menu link</a>
+		<a class="menu-link">menu link</a>
+		<a class="menu-link">menu link</a>
```
Comment (MarcFairbrother) [inline stories/qa/menu/menu.stories.html:9]: These links could be grouped into a list.

File: stories/scss/menu/menu.stories.html [hunk 3]
```diff
@@ -1,12 +1,18 @@
-<nav class="menu {{mod}} {{palette}}">
+<nav class="menu {{mod}}" [class.mod-noBorder]="border === false">
 	<a
-		type="button"
+		role="button"
```
Comment (MarcFairbrother) [inline stories/scss/menu/menu.stories.html:3]: Why not use a `<button>` instead of an `<a>`?

File: packages/scss/src/components/_menu.scss [hunk 4]
```diff
@@ -1,84 +1,85 @@
...
-		content: "";
+	&:not(.mod-noBorder)::after {
+		content: '';
 		position: absolute;
 		bottom: 0;
 		height: 1px;
 		left: 0;
 		right: 0;
-		border-bottom: 1px solid _theme("commons.divider.color");
+		border-bottom: 1px solid _theme('commons.divider.color');
```
Comment (vvalentin-lucca) [inline packages/scss/src/components/_menu.scss:16]:
```suggestion
		border-bottom: _theme('commons.divider.width') solid _theme('commons.divider.color');
```

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1423

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1423

File: packages/scss/src/components/_menu.scss (hunk 4 above)
Comment (vvalentin-lucca) [reply packages/scss/src/components/_menu.scss:16]: Non ?

**Review by MarcFairbrother — DISMISSED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1423

**Review by vvalentin-lucca — APPROVED** [review]

**Pull request merged**